src/components/ItemList/ItemList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import "../ItemMenu/ItemMenu.css"
+
+function ItemList(props) {
+
+  const menuItems = [
+    {
+      name: "Home",
+      id:''
+    },
+    {
+      name: "About Us",
+      id:''
+    },
+    {
+      name: "Services",
+      id:''
+    },
+    {
+      name: "Contacts",
+      id:''
+    }
+
+  ];
+
+  let key= 0;
+
+   function handleClick(e) {
+    e.preventDefault();
+    console.log('jkjkjk');
+  }
+
+  return(
+    <div className="header-menu">
+      {
+        menuItems.map(el => {
+          return(
+            <p className="menu-item" key={key++} onClick={handleClick}> {el.name}</p>
```

Comment:
it should be a component like <Item /> with some props, but onClick handler should be inside Item